CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 * **Interface Update!** `PolymerProject.analyzer` is no longer a required step in your build pipeline. Instead, analysis happens automatically while it fills the `project.sources()` and `project.dependencies()` streams with your project files. See [the README](/README.md) for updated examples of what build streams look like without the analyzer.
 * `StreamAnalyzer` is now `BuildAnalyzer` (since it is no longer a stream). This change only affects uses who are importing and/or using `StreamAnalyzer` directly from the `polymer-build` module.
+* **[`merge-stream`](https://github.com/grncdr/merge-stream) users:** Update to v1.0.1 or later if you are using `merge-stream` with `poylmer-build`. Stream errors do not propagate properly in previous versions of the library, and your build task may silently fail as a result.
```

Comment:
Dumb question, but what does it mean to propagate an error down a stream like this? Is this like a node event emitter `error` event? Or are we sending down `Warning` objects, or string representations of them or…